app/transformers/capd-services-api.js

```diff
@@ -0,0 +1,7 @@
+export function transformCapdPersonSummaryToAuthorisedBusinesses (summary) {
+  return summary.map((business) => ({
+    id: business.id,
+    name: business.name,
+    sbi: business.sbi
+  }))
+}
```

Comment:
We agreed were going to add unit test to cover the transformers moving forwards, would you mind adding them here?